tests/test_core.py

```diff
@@ -2322,6 +2322,12 @@ def test_pthread_equal(self):
   def test_pthread_dispatch_after_exit(self):
     self.do_run_in_out_file_test('pthread/test_pthread_dispatch_after_exit.c', interleaved_output=False)
 
+  @node_pthreads
+  def test_pthread_atexit(self):
```

Comment:
Can we add a comment either here or on the C source file describing exactly what we are testing?

Reply:
Good idea! Done with commit 786473b83591d39792dcb197bc8add00692d57a7.